parastell/invessel_build.py

```diff
@@ -761,6 +763,8 @@ def generate_invessel_build():
 
     vmec_file, invessel_build_dict = read_yaml_config(args.filename)
 
+    logger = log.check_init(None, null_logger=False)
```

Comment:
I'm not sure we want to use `check_init` here.  This is our script and we can actively decide whether we want a logger or not, perhaps based on a parser argument.

Reply:
Specifically, we can call `log.init()` since we know we want a logger instead of `log.check_init()` with arguments that force a logger to be created.